backend/api/Areas/Tools/Helpers/ImportPropertiesHelper.cs

```diff
@@ -234,84 +232,6 @@ private Entity.Agency GetOrCreateAgency(Model.ImportPropertyModel property)
             return agency;
         }
 
-        /// <summary>
-        /// Get or create a new city for the specified code.
-        /// This is a recursive function so that it can generate a new city with a unique code.
-        /// It has a limitation of which the code must be 4 characters and therefore it can only handle 9 variations (i.e. BRe1, BRe2...)
-        /// </summary>
-        /// <param name="code"></param>
-        /// <param name="name"></param>
-        /// <param name="attempts"></param>
-        /// <returns></returns>
-        private Entity.City GetOrCreateCity(string code, string name, int attempts = 1)
```

Comment:
We don't want to create cities anymore.  We want administrative areas to come from BC Data Warehouse.  This functionality will need to be added.